config/.env

```diff
@@ -1,2 +1,3 @@
-VERSION=0.0.1
-DEFAULT_OPENNMS_VALUES_FILE=./charts/opennms/values.yaml
\ No newline at end of file
+VERSION=0.0.2
+DEFAULT_OPENNMS_VALUES_FILE=./charts/opennms/values.yaml
+IMAGE_UPDATE_FREQUENCY=60
```

Comment:
Maybe another comment here as this looks to set the same timer

Reply:
Only the above should need a comment, as it's what's setting this value in the actual runtine ENV